doc/source/install.rst

```diff
@@ -90,9 +90,14 @@ as well::
 .. warning::
 
     When using pip to install GeoPandas, you need to make sure that all dependencies are
-    installed correctly. `shapely`_ and `fiona`_ provide binary wheels with the
-    dependencies included for Mac and Linux, but not for Windows.
-    `pyproj`_ and `rtree`_ don't provide those.
+    installed correctly.
+
+    - `shapely`_ and `fiona`_ provide binary wheels with the
+      dependencies included for Mac and Linux, but not for Windows.
+    - `pyproj`_ 2.x provides binary wheels with depencies included
+      for Mac, Linux, and Windows. However `pyproj`_ 1.x does not provide wheels for
+      any platform.
+    - `rtree`_ does not provide those.
```

Comment:
Can you add a blank line after this one?

Reply:
Sure, just updated.